### ISSUE TITLE
hdiutil fails when using debug=1 on Monterey (running installomator.sh from ~/Downloads)

### ISSUE THREAD

**Issue by mjsanders** [Installomator/Installomator#203]
I noticed that Installomator fails on Monterey with Debug=1, running the script from ~/Downloads.
It seems Monterey has more checks for mounting dmg's, it fails from ~/Downloads, but not from /var/folders...
Running Installomator with sudo.

the error is (sorry, some Dutch here)

```
2021-07-08 10:01:07 googlechrome Mounting Downloads/Google Chrome.dmg
hdiutil: attach failed - Bestand of directory bestaat niet
2021-07-08 10:01:07 googlechrome Error mounting Downloads/Google Chrome.dmg
```

With Debug=0 a tmpdir is created, and the mounting works as expected:

```
2021-07-08 10:05:08 googlechrome Mounting /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.VoGKMY3Q/Google Chrome.dmg
2021-07-08 10:05:18 googlechrome Mounted: /Volumes/Google Chrome
2021-07-08 10:05:18 googlechrome Verifying: /Volumes/Google Chrome/Google Chrome.app
2021-07-08 10:05:33 googlechrome Team ID matching: EQHXZ8M8AV (expected: EQHXZ8M8AV )
```

Tested with Google Chrome and Slack.
Monterey beta 2, on M1 MacBook Pro, master branch.

I am new to this team, so not sure how to proceed. Can provide more logs if needed.

**Comment by Theile** [comment]
Can you try to give /bin/zsh access to Downloads folder. I would think that should make it work. Otherwise test it from another folder.

**Comment by mjsanders** [comment]
Good catch, @Theile, these are things one can forget on Beta testing devices. 
I have checked, 'Terminal' has access to ~/Downloads.
I have added "Terminal' and `/bin/zsh` to 'Full Disk Access' and same error keeps repeating.

Luckily, it can mount the .dmg when I run the script from `/Users/Shared/` both for `debug=0` and `debug=1`
Note: with `debug=1` there is nothing copied to /Applications/, but that is expected behavior

Consider this issue closed
